core/src/main/scala/scalapb/zio_grpc/GIO.scala

```diff
@@ -1,11 +1,14 @@
 package scalapb.zio_grpc
 
-import zio.Task
+import zio.{Task, ZIO}
 import io.grpc.Status
 
 object GIO {
   def fromTask[A](task: Task[A]) =
     task.mapError(e => Status.INTERNAL.withDescription(e.getMessage).withCause(e))
 
-  def effect[A](effect: => A) = fromTask(Task.effect(effect))
+  @deprecated("use attempt", "0.6.0")
```

Comment:
Not sure what to put here for the next version of zio-grpc that will be released with zio 2.0.0